cypress/integration/BounceRulesContainer/bounce_rules.spec.js

```diff
@@ -59,21 +61,28 @@ describe("Bounce Rules Page", () => {
     });
   });
 
-  it("should cancel creating a bounce rule after submitting", () => {
-    BounceRulesPage.deleteBounceRuleAPI(testCreateRule).then(() => {
-      BounceRulesPage.open();
-      BounceRulesPage.createRuleButton.click();
-      BounceRulesPage.fillCreateRuleForm(neverCreatedRule);
-      BounceRulesPage.cancelCreateConfirmationSubmit.click();
-      BounceRulesPage.confirmModal.should("not.be.visible");
+  it("should delete a bounce rule", () => {
+    BounceRulesPage.deleteBounceRuleAPI(testDeleteRule).then(() => {
+      BounceRulesPage.createBounceRuleAPI(testDeleteRule).then(() => {
+        BounceRulesPage.open();
+        BounceRulesPage.deleteBounceRuleUI(testDeleteRule);
+        BounceRulesPage.fillCommitUI(commitMessage);
+        BounceRulesPage.deleteConfirmationConfirm.click();
+        BounceRulesPage.testBounceRuleToDelete.should("not.be.visible");
+      });
     });
   });
 
-  it("should delete a bounce rule", () => {
-    BounceRulesPage.createBounceRuleAPI(testDeleteRule).then(() => {
-      BounceRulesPage.open();
-      BounceRulesPage.deleteBounceRuleUI(testDeleteRule);
-      BounceRulesPage.testBounceRuleToDelete.should("not.be.visible");
-    });
+  it("should show alert if commit is empty", () => {
+    BounceRulesPage.deleteBounceRuleAPI(testDeleteRuleFailed)
+      .wait(1000)
```

Comment:
I'm assuming the backend is keeping that restriction the same with the keys and you have to wait 1000 still? Why are we waiting here?

Reply:
Don't think the backend has updated that portion yet, but will fix as soon as they do.